planners:
  - name: 'SplinedVoronoiPlanner'
    type: 'splined_voronoi/SplinedVoronoiPlanner'

SplinedVoronoiPlanner:
  free_cell_threshold: 0
  angle_threshold: 0.0 # 0.3927 # PI / 8; 22,5Â°
  min_distance_control_points: 1.25
  plan_resolution: 50 # pt / m
  max_curvature: 40  # value is used unless it is -1, then max curvature is calculated from formation radius.
  curvature_safety_margin: 0.02
  free_space_factor: 2.3
  optimize_lengths: false
  max_optimization_time: 4.0
  perform_splining: true
  fast_mode: false # if true the existing VD will not be updated. This is faster but may result in collisions.

  formation_config:
    robot_names: ["/mir1", "/mir2"]
    mir1:
      namespace: /mir1
      rel_x_offset: -6.0
      rel_y_offset: 1.5.0
      rel_yaw_offset: 0.0
    mir2:
      namespace: /mir2
      rel_x_offset: -3.0
      rel_y_offset: -1.5.0
      rel_yaw_offset: 0.0
